app/controllers/admin/schedules_controller.rb

```diff
@@ -9,7 +9,10 @@ class SchedulesController < Admin::BaseController
     load_resource :selected_schedule, through: :program, singleton: true
     load_resource :venue, through: :conference, singleton: true
```

Comment:
This line defines @venue variable, you can read more about it at https://github.com/CanCanCommunity/cancancan/wiki/Authorizing-controller-actions#choosing-actions